rpcs3/rpcs3qt/gui_application.cpp

```diff
@@ -451,6 +451,21 @@ void gui_application::StopPlaytime()
 */
 void gui_application::OnChangeStyleSheetRequest()
 {
+	const QString config_dir = qstr(fs::get_config_dir());
+
+	// Remove old fonts
+	QFontDatabase::removeAllApplicationFonts();
+
+	// Add PS3 fonts
+	QDirIterator ps3_font_it(qstr(g_cfg.vfs.get_dev_flash() + "data/font/"), QStringList() << "*.ttf", QDir::Files, QDirIterator::Subdirectories);
```

Comment:
why did you move it all up ?
none of these fonts are used if we don't actually load a stylesheet

Reply:
Fixed now?

Reply:
what are you even doing???